examples/near_global_ocean_simulation.jl

```diff
@@ -67,10 +67,10 @@ grid = ImmersedBoundaryGrid(grid, GridFittedBottom(bottom_height))
 bathymetry = deepcopy(Array(interior(bottom_height, :, :, 1)))
 bathymetry[bathymetry .>= 0] .= NaN
 
-fig = Figure(size = (800, 400))
-ax = Axis(fig[1, 1], title = "Bathymetry [m]")
-hm = heatmap!(axis, bathymetry, colormap = :deep, colorrange = (-6000, 0))
-cb = Colorbar(fig[1, 2], hm)
+fig = Figure(size = (1200, 400))
+ax  = Axis(fig[1, 1])
+hm = heatmap!(ax, bathymetry, colormap = :deep, colorrange = (-6000, 0))
+cb = Colorbar(fig[0, 1], hm, label = "Bottom height [m]", vertical = false)
```

Comment:
```suggestion
cb = Colorbar(fig[0, 1], hm, label = "Bottom height (m)", vertical = false)
```

We use this style for units throughout the docs here and in Oceananigans. We can change it though. I just think we should use the same style throughout.